packages/init/__tests__/logInOrBypass.test.ts

```diff
@@ -134,20 +98,12 @@ describe("loginOrBypass", () => {
         lastName,
       });
 
-    const result = await loginOrBypass(fakeBase);
+    const result = await loginOrBypass();
```

Comment:
It's the application Mode, when we initialise the client.
This way we have pre-defined ApisEndpoints, and we don't pass a `base` string everywhere.

The base approach to me is too weak because it's a string and on top of that we don't really know what it looks like :x.
I find applicationMode being more intuitive (dev / stage / prod).

Even for QA, now it will be `--mode stage` instead of `--base https://.....`